### ISSUE TITLE
When oneFingerPan is true clicks on SVG elements are not working anymore

### ISSUE THREAD

**Issue by meriturva** [svgdotjs/svg.panzoom.js#93]
is it possible to enable `oneFingerPan` and still receive mouse events from SVG elements?

Thanks

**Comment by Fuzzyma** [comment]
Mh, this is not that easy to solve. Some people want to be able to pan even if you "start" the pan on an element. You however dont want panning on elements. I am not really sure how to satisfy both

**Comment by meriturva** [comment]
I guess the only way is a sort of timeout handling and move difference between mouseDown and mouseUp event. What do you think?

**Comment by Fuzzyma** [comment]
That will complicate the code quite a bit and timeouts are really sensitive to race conditions and stuff not behaving the same all the time. Thats why I avoid them. btw: what mouse events exactly are not working for elements anymore? Afaik, we don't stop Propagation and only prevent default for touch down

**Comment by meriturva** [comment]
Seems to be blocked globally, I could prepare a simple repo to play with.....what do you think?

**Comment by Fuzzyma** [comment]
sure there is no harm in playing around and finding better solutions

**Comment by meriturva** [comment]
Here a simple stackblitz with `oneFingerPan` enabled: https://stackblitz.com/edit/js-ulebap?file=index.html

Please use a touch screen or a device to test click.

As you could see there is no way to make click works on the top black dot.

**Comment by Fuzzyma** [comment]
Works for me when doing this:

```js
  SVG('#pointB').on('click', () => {
    console.log('test');
  });
```

**Comment by meriturva** [comment]
Thanks @Fuzzyma for your time but when I try your sample on my touch screen I have the same issue. Did you try on a touch device? if yes on a desktop touch display or mobile one?

**Comment by Fuzzyma** [comment]
No. I only tried on my laptop. You can try touchstart as well tho. Sorry, I cant spend much time to figure this out atm.

**Comment by dmitriyzhuk** [comment]
@meriturva got the same issue. found any solution for this?

**Comment by meriturva** [comment]
> @meriturva got the same issue. found any solution for this?

No actually

**Comment by Quasarman** [comment]
Have the same issue is there a proper solution for this?

**Comment by phrei** [comment]
Would be nice if a threshold could be configured. Something like "if I drag my finger for 5px, then it starts to pan, otherwise it's a click".
